back/app/models/home_page.rb

```diff
@@ -36,16 +36,16 @@ class HomePage < ApplicationRecord
 
   accepts_nested_attributes_for :pinned_admin_publications, allow_destroy: true
 
-  before_validation :sanitize_top_info_section_multiloc, if: :top_info_section_enabled
```

Comment:
Why do we need to remove this condition? It looks reasonable.

Reply:
Simply because it's not there in `AppConfiguration` validation of `homepage_info_section_multiloc`.

Reply:
Ah, so we can always validate `top_info_section_multiloc`. Maybe it makes sense indeed.